src/@next/Switch/SwitchStyle.tsx

```diff
@@ -19,7 +39,9 @@ export const SwitchStyle = styled.div`
   transition: 300ms all;
 
   @media (max-width: ${Breakpoints.large}) {
-    width: 38px;
+    ${({ withIcon }: SwitchStyleProps) => ({
```

Comment:
same here we can bind a `data-with-icon` attribute here and style this accordingly

Reply:
done